src/tests/logic.R

```diff
@@ -11,4 +11,11 @@ uploadSpectrumFile <- function(wd, path = "../../../sample_files/Malawi_2018_ver
     fileUpload$setElementAttribute("style", "display: inline")
     enterText(fileUpload, normalizePath(path))
     fileUpload$sendKeysToElement(list(key = "enter"))
+}
+
+uploadDigestFile <- function(wd, path = "../../../sample_files/testing1234.zip.shiny90") {
+    fileUpload <- wd$findElement("css", "#digestUpload")
+    fileUpload$setElementAttribute("style", "display: inline")
+    enterText(fileUpload, normalizePath(path))
+    fileUpload$sendKeysToElement(list(key = "enter"))
```

Comment:
Maybe we could factor out the common logic between this and `uploadSpectrumFile` into a helper function?